src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -54,6 +56,46 @@ public static Feature parseFeature(String featureName) throws ParseException {
         return new Feature(featureName);
     }
 
+    /**
+     * Parses {@code date} into a {@code AthletickDate} and returns it. Leading and trailing
+     * whitespaces will be trimmed.
+     * @throws ParseException if the specified date is invalid (not length of 6 or 8).
+     */
+    public static AthletickDate parseDate(String date) throws ParseException {
```

Comment:
Consider abstracting according to `trimmedDate` length as this is a very long method.

Reply:
agreed, will do so.